src/hb-ot-color-colr-table.hh

```diff
@@ -112,6 +115,38 @@ struct COLR
     return glyph_layers.length;
   }
 
+  struct accelerator_t
```

Comment:
I think you can drop this accelerator_t here.

Reply:
I was using it as a RAII style constructing the table to use `closure_glyphs ()`. I could add a helper instead, but all the other tables that add closure glyphs have an accelerator to do so...